example/generator/plugin.js

```diff
@@ -61,6 +63,25 @@ define([
 
         openmct.telemetry.addProvider(new StateGeneratorProvider());
 
+        openmct.types.addType("example.spectral-generator", {
+            name: "Spectral Generator",
+            description: "For development use. Generates example streaming telemetry data using a simple sine wave algorithm.",
+            cssClass: "icon-generator-telemetry",
+            creatable: true,
+            initialize: function (object) {
+                object.telemetry = {
+                    period: 10,
+                    amplitude: 1,
+                    wavelength: 1,
```

Comment:
do we need frequency here too, or is it just derived?

Reply:
Good point. Let's leave it at Wavelength for now. We won't be able to use Spectral Generator based on frequency, but I think that's ok.